database.cc

```diff
@@ -1268,11 +1260,7 @@ std::vector<view_ptr> keyspace_metadata::views() const {
 }
 
 schema_ptr database::find_schema(const sstring& ks_name, const sstring& cf_name) const {
-    try {
-        return find_schema(find_uuid(ks_name, cf_name));
-    } catch (std::out_of_range&) {
-        std::throw_with_nested(no_such_column_family(ks_name, cf_name));
-    }
+    return find_schema(find_uuid(ks_name, cf_name));
```

Comment:
`find_schema(uuid)` calls `find_column_family(uuid).schema()` so it will throw `no_such_column_family(uuid)` in this case.  Ditto as above, not ideal, but would work.

Reply:
Done in #9755